### PR TITLE
Review of #42 with a few code changes and added some tests

### PR DESCRIPTION
**Pull request by Scorpoon**
Review for #42:
Adopted code changes from @kroimon and @ThomDietrich, made some changes to _common, _fixtures and sentences and added some tests for hopefully every combination.

### CODE REVIEW

**Comment by synesthesiam** [comment]
@Scorpoon Looks like just a few duplicate test sentences that need to get cleaned up :+1:

**Comment by Scorpoon** [comment]
> @Scorpoon Looks like just a few duplicate test sentences that need to get cleaned up :+1: 

Saw the doc folder too late for the lint script explanation, should work now 👍

**Comment by synesthesiam** [comment]
Awesome, thanks @Scorpoon!

**Review by mib1185 — COMMENTED** [review]
Hi @Scorpoon 
great contributiuon 👍 but stil have some comments left 🙈

File: sentences/de/fan_HassTurnOff.yaml [hunk 1]
```diff
@@ -3,8 +3,19 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "(schalte | stoppe) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_off>]"
-          - "(schalte | stoppe) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
+          - "schalte <fan_variants> <area> <turn_off>"
+          - "schalte <area> <fan_variants> <turn_off>"
+          - "stoppe <fan_variants> <area> [<turn_off>]"
```
[the remark this answers is not in the document]
Comment (Scorpoon) [reply sentences/de/fan_HassTurnOff.yaml:8]: I had that in first place, don't know why i changed it back xD
Will change it and adjust the tests accordingly when i'm back from work.

**Comment by Scorpoon** [comment]
Do i need write permissions to adopt suggestions? @synesthesiam

**Comment by mib1185** [comment]
You can't apply suggestions on an merged PR 😉 
You need to create a new one

**Pull request merged**